src/main/java/Model/Database/Interaction/I_ControllerUnit.java

```diff
@@ -79,6 +79,58 @@ public static T_ControllerUnit retrieve(Connection conn, PreparedStatement ps, R
         return tc;
     }
 
+    public static ArrayList<T_ControllerUnit> retrieveFilteredAll(Connection conn, PreparedStatement ps, ResultSet rs, int flatId, int centralUnitId) throws SQLException {
+
+        // No Filter is being used
+        if (flatId <= 0 && centralUnitId <= 0) {
```

Comment:
tu treba dat nejaky komentar alebo niekde inde zapisat ze hodnota -1 je pre nefiltrovane vysledky/ziadny filter, len neviem naco je to potom v tejto metode a preco sa inde nezavolat proste miesto retrieveFilteredAll s -1, co su nefiltrovane vysledky proste len retrieveAll

Reply:
Nestane sa ze by to niekto zavolal s -1 a -1 ale dobra prax je, ze ak to vies poslat niekam tak to tam posli.